documentation/Functions.md

```diff
@@ -35,13 +35,27 @@ installDevonfwIde(["java","mvn"], "2020.08.001")
 ***
 
 ### restoreDevonfwIde <a name="restoreDevonfwIde"></a>
+It is similar to [installDevonfwIde](#installDevonfwIde) function but the difference between the two functions is that restoreDevonfwIde installs the ide through a script in the background and the Katacoda user does not have to download and start the installation.
```

Comment:
Change the end of the sentence to: the Katacoda user will already have an installed version of devonfw-ide at the start of the Tutorial.